pytest_asyncio/plugin.py

```diff
@@ -169,8 +169,13 @@ def pytest_fixture_post_finalizer(fixturedef, request):
     """Called after fixture teardown"""
     if fixturedef.argname == "event_loop":
         policy = asyncio.get_event_loop_policy()
-        # Clean up existing loop to avoid ResourceWarnings
-        policy.get_event_loop().close()
+        try:
+            loop = policy.get_event_loop()
+        except RuntimeError:
+            loop = None
```

Comment:
Would it make sense to emit a warning here? After all, if `get_event_loop` fails it means that someone was messing with the loop provided by the `event_loop` fixture. A warning would at least tell the users they are going down a dangerous path.